### PR TITLE
fix: approval pending ux

### PR DESCRIPTION
**Pull request by zzmp**
- Fixes color values so that the icons for token approval are visible
- Transitions to the Loader icon immediately upon clicking the Approve button, without waiting for wallet interaction

### CODE REVIEW

File: src/pages/Swap/index.tsx
Comment (vm) [inline src/pages/Swap/index.tsx:723]: is approvalState === PENDING still necessary? can it ever be true while approvalPending is false?

**Review by vm — APPROVED** [review]

File: src/pages/Swap/index.tsx
Comment (zzmp) [reply src/pages/Swap/index.tsx:723]: I think it can if you continue to use the UX while waiting for the pending approval to go through

**Pull request merged**
